src/test/dynamo/Beyonce.test.ts

```diff
@@ -90,20 +89,24 @@ describe("Beyonce", () => {
         })
     }))
 
-    const db = new Beyonce(table, new DynamoDB({ region: "us-west-2" }))
-    ;(db as any).client.get = mockGet
+    const db = new Beyonce(table, createDynamoDB())
+    ;(db as any).client.send = mockGet
 
     await db.get(MusicianModel.key({ id: musician.id }), {
       consistentRead: true
     })
-    expect(mockGet).toHaveBeenCalledWith({
-      TableName: table.tableName,
-      Key: {
-        pk: "musician-1",
-        sk: "musician-1"
-      },
-      ConsistentRead: true
-    })
+    expect(mockGet).toHaveBeenCalledWith(
+      expect.objectContaining({
```

Comment:
We use `objectContaining` now since there are various other properties on the argument passed to `send()` now.